docs/api/client/hoodie.rst

```diff
@@ -28,10 +28,10 @@ id, session or the connection status to the backend.
 
 Hoodie integrates Hoodie’s client core modules:
 
--  `The account API <client/hoodie.account.html>`__
--  `The store API <client/hoodie.store.html>`__
--  `The connectionStatus API <client/hoodie.connection-status>`__
--  `The log API <client/hoodie.log.html>`__
```

Comment:
> these links currently work for me on http://docs.hood.ie/en/latest/api/index.html – are you sure we can change them?

This page have proper link, but this one does not (it is one level deeper):
http://docs.hood.ie/en/latest/api/client/hoodie.html